docs/Gemfile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+source "https://rubygems.org"
+
+gem "webrick"
+gem "github-pages", "~> 226", group: :jekyll_plugins
+
```

Comment:
nit: Could you remove the extra newlines at the end of Gemfile, _config.yml, and debug_docs.md?